scripts/docker/buildpack-deps/Dockerfile.ubuntu.clang.ossfuzz

```diff
@@ -22,7 +22,7 @@
 # (c) 2016-2021 solidity contributors.
 #------------------------------------------------------------------------------
 FROM gcr.io/oss-fuzz-base/base-clang:latest as base
-LABEL version="23"
+LABEL version="1"
```

Comment:
just saw this.. `1` is really correct?

Reply:
Yes, the file was renamed incorrectly in the previous PR dealing with this - and also the version should already have been reset to 1 there.

Reply:
So it's now the first image we actually build under the new name.

Reply:
ah oki!